test/1_stdlib/ArrayCore.swift

```diff
@@ -21,7 +21,7 @@ import Swift
 var trackedCount = 0
 var nextTrackedSerialNumber = 0
 
-final class Tracked : ForwardIndex, CustomStringConvertible {
+final class Tracked : Equatable, CustomStringConvertible {
```

Comment:
Well actually I am stepping out for a while so if you feel the need to grab these changes go ahead an merge otherwise I will work more on it and likely squash the commits.